### ISSUE TITLE
ExampleCount metric always returns 1 for multi-output models

### ISSUE THREAD

**Issue by rafaelbarreto87** [tensorflow/model-analysis#62]
### System information

-   **Have I written custom code (as opposed to using a stock example script
    provided in TensorFlow Model Analysis)**: No
-   **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Mac OS X
-   **TensorFlow Model Analysis installed from (source or binary)**: binary
-   **TensorFlow Model Analysis version (use command below)**: 0.21.6
-   **Python version**: Python 3.6.9
-   **Jupyter Notebook version**: doesn't apply
-   **Exact command to reproduce**:

### Describe the problem

`tfma.metrics.ExampleCount` always returns 1 for multi-output models. I believe this happens because when the metric key is created, the output name is not considered (see https://github.com/tensorflow/model-analysis/blob/master/tensorflow_model_analysis/metrics/example_count.py#L49). This is not a problem in the similar `tfma.metrics.WeightedExampleCount` (https://github.com/tensorflow/model-analysis/blob/master/tensorflow_model_analysis/metrics/weighted_example_count.py#L60).

### Source code / logs

```
eval_config = tfma.EvalConfig(
    model_specs=[
        tfma.ModelSpec(
            prediction_key='softmax',
            signature_name="eval",
            label_key="candidate_app_id_integerized"
        )
    ],
    metrics_specs=tfma.metrics.specs_from_metrics(
        metrics=[
            tfma.metrics.ExampleCount(), 
            tfma.metrics.WeightedExampleCount(),
            tf.keras.metrics.SparseCategoricalCrossentropy()
        ],
        output_names=['softmax'],
        include_example_count=False,
        include_weighted_example_count=False
    ),
    slicing_specs=[
        tfma.SlicingSpec(),
        tfma.SlicingSpec(
            feature_keys=["language"]
        ),
        tfma.SlicingSpec(
            feature_keys=["country_code"]
        ),
        tfma.SlicingSpec(
            feature_keys=["is_ios"]
        )
    ]
)

eval_result = tfma.run_model_analysis(
    eval_shared_model=eval_shared_model,
    eval_config=eval_config,
    data_location=examples_dir,
    output_path=output_path
)
```

**Comment by mdreves** [comment]
This is a display issue. The example count is the same for all outputs, so it should be filtered out of the choices for the drop down.

**Comment by rafaelbarreto87** [comment]
Right. This makes total sense.

**Comment by UsharaniPagadala** [comment]
@rafaelbarreto87 

Closing this issue as it is resolved,feel free to reopen if this still exists.Thanks
